.github/workflows/ci.yaml

```
@@ -72,7 +72,7 @@ jobs:
         uses: actions/cache@d4323d4df104b026a6aa633fdb11d772146be0bf # v4
         with:
           path: ${{env.ucm_local_bin}}
-          key: ucm-${{ matrix.os }}-${{ hashFiles('**/stack.yaml', '**/package.yaml', '**/*.hs', '**/unison-cli-integration/integrationtests/IntegrationTests/*')}}
+          key: ucm-${{ matrix.os }}-${{ hashFiles('**/ci.yaml', '**/stack.yaml', '**/package.yaml', '**/*.hs', '**/unison-cli-integration/integrationtests/IntegrationTests/*')}}
```

So, the key for the “restore binaries” step needs to match this one, but note that before my change it’s already missing the integration tests files:

https://github.com/unisonweb/unison/blob/0d7f0222e8c65d99a4417ff9dc7aab886c392537/.github/workflows/ci.yaml#L235-L241

That didn’t break anything because there’s a typo (it’s missing a hyphen):

```suggestion
          key: ucm-${{ matrix.os }}-${{ hashFiles('**/ci.yaml', '**/stack.yaml', '**/package.yaml', '**/*.hs', '**/unison-cli-integration/integration-tests/IntegrationTests/*')}}
```